packages/zone/src/durable.js

```diff
@@ -89,6 +89,10 @@ export const makeDurableZone = (baggage, baseLabel = 'durableZone') => {
   /** @type {import('.').Zone['exoClass']} */
   const exoClass = (...args) => prepareExoClass(baggage, ...args);
   /** @type {import('.').Zone['exoClassKit']} */
+  // @ts-ignore This type check regressed inexplicably with the release
```

Comment:
I see lint gave you the expected second order error. I have suppressed this either with another @ts-ignore or an eslint-disable-next-line . I forget which. In any case, I suspect plenty of these remain in the code.